CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v1.18.0
+------------------------------
+*November 29, 2018*
```

Comment:
This will need updating, but I'd wait until you have to merge in `v1.17.0`

Reply:
Updated